src/contracts/periphery/Operator.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import "./RolesAuthority.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/interfaces/IERC1271.sol";
+
+
+contract Operator is RolesAuthority {
+    constructor(address _owner) RolesAuthority(_owner) {}
+
+    function multicall(address[] memory targets, bytes[] memory data) external returns (bytes[] memory results) {
```

Comment:
How does this impact ABI legibility and transaction construction? As a client, I now need to consume two ABIs to make a single call, and manually construct/encode calldata, no?

Reply:
I also will need to, for a specific delegation, find the correct operator factory to multi-call through. How do I do this if my account can be a delegate of multiple operators? Or are we adding the 1:1 restriction back in?

And how does this approach different from using a module as part of a multi-sig?

Reply:
> I also will need to, for a specific delegation, find the correct operator factory to multi-call through. How do I do this if my account can be a delegate of multiple operators?

Need to run an indexer for knowing all operators you can act on behalf of. Same problem exists in uniform delegation PR, unless we add additional storage. This isn't a requirement in PRD